packages/client/src/__tests__/integration/happy/exhaustive-schema-mongo/__snapshots__/test.ts.snap

```diff
@@ -341,39 +341,39 @@ import $Extensions = runtime.Types.Extensions
 export type PrismaPromise<T> = $Public.PrismaPromise<T>
 
 
-export type EmbedPayload<ExtArgs extends $Extensions.Args = $Extensions.DefaultArgs> = {
```

Comment:
I think this was added so we can later support composite result extensions.

Reply:
We could always add them back once we get to it - it's a matter of removing 2 if statements in generator. I just felt that since we are not using them at the moment it does not make sense to generate `ExtArgs` for composites.

Reply:
Could you please create a placeholder or issue in our private board? (we've got quite a few of these things we want to add later)